helion/autotuner/config_spec.py

```diff
@@ -252,17 +255,29 @@ def normalize(self, config: helion.Config | dict[str, object]) -> None:
 
         # TODO(jansel): include num_ctas and max_nreg
 
-        for name, values in (("pid_type", VALID_PID_TYPES),):
+        for name, values in (
+            ("pid_type", VALID_PID_TYPES),
+            ("num_sm_multiplier", VALID_NUM_SM_MULTIPLIERS),
+        ):
             if name in config:
                 if config[name] not in values:
                     raise InvalidConfig(
-                        f"Invalid value for {name!r}: {config[name]!r} must be one of {[*values]!r}"
+                        f"Invalid value for {name!r}: {config[name]!r} must be one of {list(values)!r}"
                     )
             else:
                 config[name] = values[0]
 
-        # Set default values for grid indices when pid_type is not persistent
+        # Validate num_sm_multiplier is only used with persistent kernels
         pid_type = config["pid_type"]
+        num_sm_multiplier = config.get("num_sm_multiplier", 1)
+        if num_sm_multiplier != 1 and pid_type in ("flat", "xyz"):
```

Comment:
we'd probaby also want to exclude values larger than 1 if careveout/reserved sm is set